workspaces/quickstart/plugins/quickstart/src/components/QuickstartDrawer.tsx

```diff
@@ -30,6 +31,12 @@ export const QuickstartDrawer = () => {
   const quickstartItems: QuickstartItemData[] = config?.has('app.quickstart')
     ? config.get('app.quickstart')
     : [];
+
+  // This will be dynamically determined based on the logged-in user
+  const userRole = 'developer'; // switch to 'admin', 'developer', or other roles for testing purposes
```

Comment:
Do you plan to update this later to use roles dynamically?

Reply:
I would expect it to be completed in the same PR or story. Going ahead with a hardcoded value like that doesn't make sense to me.